src/components/common/ProfilePanel.js

```diff
@@ -20,6 +19,7 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import "styles/Panel.scss";
 import Footer from "./Footer";
 
+import ImageSelector from "../Sketches/components/ImageSelector"
```

Comment:
I think this import statement is just a little awkward (since we're importing from going down). I think it might make more sense to put the `ImageSelector.js` file in `src/components/common` - what do you think?